### ISSUE TITLE
Question about Kuiper test

### ISSUE THREAD

**Issue by A-ep93** [circstat/pycircstat#74]
I want to use the code for kuiper test but my data set has a large number of arrays (about 2000). I can see that the maximum value of "N" is 501 in your code (kuiper_table.npy). How should I modify this table for larger numbers? Do you have any references for these values that can share with me?
In the threads, I read that I can use the value of the largest N in the code (501) for all larger sizes of arrays. Is this a safe choice for both cases when the hypothesis gets rejected/accepted?

**Comment by philippberens** [comment]
Yes, the p-value is then larger than it would need to be, but this is typically not an issue.
